### PR TITLE
'Access collection' tooltip doesn't show collection name

### PR DESCRIPTION
**Pull request by uyscuti-wiki**
[T254362](https://phabricator.wikimedia.org/T254362)

### CODE REVIEW

**Comment by Samwalton9** [comment]
This needs rebasing/repointing at the master branch.

**Comment by Samwalton9** [comment]
That's odd, I'm not seeing the enlarged tooltip on Chrome, which browser are you using?

**Comment by suecarmol** [comment]
> That's odd, I'm not seeing the enlarged tooltip on Chrome, which browser are you using?

I am using Firefox. Maybe I can take a look at this today to see if this is fixable on FF.

File: TWLight/users/templates/users/resource_tile.html [hunk 1]
```diff
@@ -32,28 +32,56 @@
...
+              {% endif %}
             {% else %}
-              {% comment %}Translators: Title for logo in tiles that are linked to partner description  page. Don't translate {{ name }}. {% endcomment %}
-              {% blocktrans trimmed with resource.name as name %}
-                 Link to {{ name }} signup page
-              {% endblocktrans %}
+              {% if resource.stream %}
+                {% comment %}Translators: Title for logo in tiles that are linked to partner description  page. Don't translate {{ name }}.{% endcomment %}
+                {% blocktrans trimmed with name=resource.stream %}
+                   Link to {{ name }} signup page
```
Comment (suecarmol) [inline TWLight/users/templates/users/resource_tile.html:50]: This should have an `'s` like the content above

File: TWLight/users/templates/users/resource_tile.html [hunk 2]
```diff
@@ -32,28 +32,56 @@
...
+                {% endblocktrans %}
+              {% else %}
+                {% comment %}Translators: Title for logo in tiles that are linked to partner description  page. Don't translate {{ name }}.{% endcomment %}
+                {% blocktrans trimmed with name=resource.partner %}
+                   Link to {{ name }} signup page
```
Comment (suecarmol) [inline TWLight/users/templates/users/resource_tile.html:55]: Same here

**Pull request merged**